tests/testsuite/config.rs

```diff
@@ -115,17 +117,17 @@ impl ConfigBuilder {
             &self.unstable,
             &self.config_args,
         )?;
-        Ok(config)
+        Ok(gctx)
     }
 }
 
-fn new_config() -> Config {
-    ConfigBuilder::new().build()
+fn gctx() -> GlobalContext {
```

Comment:
I think I like `new` being in the name as it conveys a new instance is being created, rather than accessing an existing instance (as most people will be looking at callers and might not notice this moves the value)